src/appkit.cs

```diff
@@ -2125,6 +2125,10 @@ public interface NSButton : NSAccessibilityButton {
 		[Mac (10,12)]
 		[Export ("imageHugsTitle")]
 		bool ImageHugsTitle { get; set; }
+
+		[Mac (10,5)]
```

Comment:
yeah, those are always worth checking (wrench bots will do it) as they are often mistakes